connector/src/main/java/org/geysermc/connector/dump/DumpInfo.java

```diff
@@ -55,7 +56,7 @@
     private final DumpInfo.VersionInfo versionInfo;
     private Properties gitInfo;
     private final GeyserConfiguration config;
-    private final Object floodgateConfig;
+    private final Object floodgate;
```

Comment:
This can be defined as a Floodgate class since it's not a class from another project.

Reply:
Sorry, I don't think I fully understand, can you expand?

Reply:
`private final Object floodgate` -> `private final Floodgate floodgate`

Reply:
makes sense, got it